bin/docker-entrypoint

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+dockerize -wait tcp://db:5432 -timeout 20s -wait-retry-interval 10s bash -c "bin/rails db:create"
+bin/rails db:prepare
+bin/rails parallel:create
+bin/rails parallel:prepare
+bin/rails db:seed
+
+bundle exec rake assets:precompile
```

Comment:
No change necessary: This might not be needed now that you got shakapacker working?

Reply:
From what I could find it seems like that's a maybe, depending on how we're using those two tools. I'm going to leave that judgment call for when I have spent more time working in this repo!